uin_card_template.html

```diff
@@ -0,0 +1 @@
 } .name-head-color { color: black; } .head-title { margin-left: -85px; } .bar-code-padding { margin-top: 20px; margin-left: 20px; } .top-buffer { margin-left:10px; } </style> </head> <body> <table class="main-table"> <tr> <td>&nbsp; <div class="cir"><font size="1">Logo</font></div> </td> <td><label class="name-head-color"><font size="5"class=" head-title"> &nbsp;&nbsp; Kingdom of MyCountry &nbsp;&nbsp;&nbsp; </font> </label></td> <td rowspan="4"> <div> <div style="border: solid black 1px; height: 150px; width: 120px;"> #if($isPhotoSet)<img src = $!ApplicantPhoto style="width: 120px; height: 150px"> #end  </div> </div> </td> </tr> <tr> <td rowspan="1"> <div class="block top-buffer"> <label class="name-head-color"><b>UIN&nbsp;:&nbsp;</b></label> <span class="name-color"> $!UIN </span> </div> </td> </tr> <tr> <td rowspan="1"> <div class="block top-buffer"> <label class="name-head-color"><b>Name&nbsp;:&nbsp;</b></label> <span class="name-color"> $!fullName_eng</span> </div> </td> </tr> <tr> <td rowspan="1"> <div class="block top-buffer"> <label class="name-head-color"><b>اسم&nbsp;:&nbsp;</b></label> <span>$!fullName_ara</span> </div> </td> </tr> <tr> <td rowspan="1"> <div class="block top-buffer"> <label class="name-head-color"><b>DOB/تاريخ&nbsp;:&nbsp;</b></label> <span>$!dateOfBirth</span> </div> </td> </tr> <tr> <td colspan="2"> <div class="block top-buffer"> <label class="name-head-color"><b>Gender&nbsp;:&nbsp;</b></label> <span>$!gender_eng</span> &nbsp;&nbsp; <label class="name-head-color"><b>جنس&nbsp;:&nbsp;</b></label> <span>$!gender_ara</span> </div> </td> </tr> <tr> <td colspan="3"> <div class="row" style="margin-right: 0px; margin-left: -10px;"> <div class="col-md-2 top-buffer"> <div class="block top-buffer"> <table> <tr> <td><label class="name-head-color"><b>Address&nbsp;:&nbsp;</b></label> </td> <td><span>$!addressLine1_eng, $!addressLine2_eng, $!addressLine3_eng, $!region_eng, $!province_eng, $!city_eng, $!postalCode </span></td> </tr> <tr> <td><label class="name-head-color"><b>عنوان&nbsp;:&nbsp;</b></label> </td> <td><span>$!addressLine1_ara, $!addressLine2_ara, $!addressLine3_ara, $!region_ara, $!province_ara, $!city_ara, $!postalCode </span> </td> </tr> </table> </div> </div> </div> </td> </tr> </table> <div>&nbsp;</div> <table class="main-table" style="height: 300px"> <tr> <td> <div style="margin-left: 10px; margin-right: 10px; border: solid black 1px; height: 250px; width: 250px;"> <div class="col-md-6"> <div class="bar-code-padding"></div> <img src=$!QrCode style="width: 250px; height: 250px; margin-top: -20px"> </div> </div> </td> <td> <div class="name-head-color col-md-6"> <br> For any issues please contact us at<br> <br> Registration Proccessor,Hanging Gardens,Global Village Tech Park, Mysore Rd,RVCE,Bengaluru, Karnataka 560059<br> <br> لأية مشاكل يرجى الاتصال بنا على <br>التسجيل المعالج، الحدائق المعلقة ، القرية العالمية Tech Park, Mysore Rd,RVCE 560059 بنغالورو، كارناتاكا <br> <br> </div> </td> </tr> </table> </body> </html>
```

Comment:
I remember Sasi mentioning about some encoding issue with plain template getting served from config service, is this config tested with multi lang support template getting serviced from config server ?

Reply:
for testing, I have added @vishwa-vyom we need to check once after the merge.